### PR TITLE
188618452 new table clean up

### PR DESCRIPTION
**Pull request by eireland**
On new tables:
- Removes "no cases" div
- Removes empty case above the input line
- Changes default attribute name to "Attribute Name" instead of "AttributeName"
- Changes table created from the `--new--` in the table menu default width to just show the one default column

### CODE REVIEW

File: v3/cypress/e2e/formula/formula-component.spec.ts [hunk 1]
```diff
@@ -83,15 +83,16 @@ context("Formula Engine", () => {
         "❌ Undefined symbol v1"
       ])
     })
-    it("Formula in a new dataset", () => {
+    // Skipped for now until insert cases on input row is implemnted
```
Comment (tealefristoe) [inline v3/cypress/e2e/formula/formula-component.spec.ts:86]: How difficult would it be to implement this? Isn't this already done  in `table.spec.ts` lines 738-744?

**Review by tealefristoe — CHANGES_REQUESTED** [review]
Overall looks good. I just had a few questions/suggestions that I think are worth addressing.

File: v3/cypress/e2e/plugin.spec.ts [hunk 2]
```diff
@@ -326,19 +332,66 @@ context("codap plugins", () => {
     webView.clearAPITesterResponses()
     // TODO Check for dragleave notification when dragging to plugin then out of plugin
 
+    // For tests involving drag and drop of components with attribute "Attribute Name",
+    // we use a different drag and drop code because the cy.command version includes a
+    // "contains" expectation of text "Attribute Name", which it cannot find because it is over
+    // two spans. This code omits the "contains" expectation because the selector already has enough
+    // information without needing to find a specific text in a span
```
[the remark this answers is not in the document]
Comment (eireland) [reply v3/cypress/e2e/plugin.spec.ts:339]: The default column width came from V2. 
I couldn't get the existing function to work with the attribute name over 2 spans. I'll modify it and add it to the table element object.

File: v3/cypress/e2e/formula/formula-component.spec.ts (hunk 1 above)
Comment (eireland) [reply v3/cypress/e2e/formula/formula-component.spec.ts:86]: Part of this PR removed the single case created when a new table is first created. 
Input row index cell does not open the index column menu so new cases cannot be inserted without having to enter data.
The `table.spec.ts` lines 738-744 tests that a new case is created at the bottom when user enters data into the input line.

**Review by kswenson — APPROVED** [review]
👍 LGTM aside from the issues @tealefristoe pointed out.

File: v3/cypress/e2e/formula/formula-component.spec.ts (hunk 1 above)
Comment (eireland) [reply v3/cypress/e2e/formula/formula-component.spec.ts]: done

**Review by tealefristoe — APPROVED** [review]
Looks good, thanks for addressing those issues 👍

**Pull request merged**